sgt-back-end/server.js

```diff
@@ -0,0 +1,171 @@
+import express from 'express';
+import pg from 'pg';
+
+const app = express();
+
+app.use(express.json());
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+app.get('/api/grades', async (req, res) => {
+  try {
+    const sql = `
+      select *
+        from "grades"
+    `;
+    const result = await db.query(sql);
+    res.status(200).json(result);
```

Comment:
This will return an object. You want to return an array. (Check out the `rows` property of the result.)